### DISCUSSION TITLE
When the inline query returns video query, it says that there is a problem with the thumbnail format verification code

### DISCUSSION THREAD

**Discussion by ZhiYiHub** [rubenlagus/TelegramBots#1238] [Q&A] [answered]
![image](https://github.com/rubenlagus/TelegramBots/assets/63048686/4a60402b-ae6b-4aad-8a38-42d4d3d3473d)
![image](https://github.com/rubenlagus/TelegramBots/assets/63048686/a3d4b6df-49d5-457f-9615-067236a53ae9)

**Comment by ZhiYiHub** [comment]
help  me！！

**Comment by rubenlagus** [comment, marked as answer]
That's fixed in later releases
